### PR TITLE
Big endian fixes

### PR DESCRIPTION
**Pull request by ondrejholy**
It fixes two failing tests on big endian. See commit descriptions. Relates to https://github.com/FreeRDP/FreeRDP/issues/4231.

### CODE REVIEW

**Comment by akallabeth** [comment]
@freerdp-bot test

File: client/common/file.c [hunk 1]
```diff
@@ -550,12 +447,12 @@ static BOOL freerdp_client_parse_rdp_file_buffer_ascii(rdpFile* file, const BYTE
 		{
 			beg = line;
 
-			if (!freerdp_client_parse_rdp_file_add_line_ascii(file, line, index))
+			if (freerdp_client_parse_rdp_file_add_line(file, line, index) == -1)
 				return FALSE;
```
Comment (akallabeth) [inline client/common/file.c]: Memoryleak ```copy```

File: client/common/file.c [hunk 2]
```diff
@@ -550,12 +447,12 @@ static BOOL freerdp_client_parse_rdp_file_buffer_ascii(rdpFile* file, const BYTE
...
 
 			if (beg[0] == '/')
 			{
-				if (!freerdp_client_parse_rdp_file_option_ascii(file, line, index))
+				if (!freerdp_client_parse_rdp_file_option(file, line, index))
 					return FALSE;
```
Comment (akallabeth) [inline client/common/file.c]: Memoryleak ```copy```

**Comment by ondrejholy** [comment]
@akallabeth Attached fix, although those are pre-existing...

**Comment by akallabeth** [comment]
@ondrejholy Yes, looks like they were there for some time...

**Pull request merged**